collector/database/settings.php

```diff
@@ -46,6 +46,16 @@
         )
     );
 
+    $ADMIN->add(
+        'reports',
+        new admin_externalpage(
+            'cltr_database_reset',
+            get_string('metric_reset', 'cltr_database'),
+            new moodle_url('/admin/tool/cloudmetrics/collector/database/reset.php'),
```

Comment:
I think this would also be more useful if linked from the chart page like the backfill, to the reset form for the correct metric

Everything else looks good

Reply:
I have added a link from backfill page